examples/repl/main.rs

```diff
@@ -460,7 +460,7 @@ fn main_0(args: Vec<String>) -> Result<(), failure::Error> {
         unsafe {
             let mut ctx = ctx.write().unwrap();
             dc_close(&mut ctx);
-            dc_context_unref(&mut ctx);
+            std::mem::drop(ctx);
```

Comment:
why is this needed?

Reply:
I would think that the above could be replaced with 

```rust
let _ = ctx.into_inner()?;
```
that consumes the inner value of the lock dropping it when it goes out of scope, triggering `dc_close`

Reply:
Whoops, didn't realise this was an Arc, will fix.

Reply:
Though both the Arc and the RwLock implement Drop, so I think this would result in the context being dropped.  But really this isn't needed at all since the whole thing goes out of scope in the next line anyway, I just thought this would be more explicit.  So if I'm changing anything I'd rather just remove the line completely.